docs/UserGuide.md

```diff
@@ -153,183 +142,158 @@ Format: `vedit INDEX [n/NAME] [p/PHONE] [e/EMAIL] [a/ADDRESS] [t/TAG]…​`
+* The index **must be a positive integer** 1, 2, 3, …
+
+Restrictions:
+* The volunteer ID must be an integer that represents a valid volunteer number in the volunteer list. If there are 30 volunteers in the volunteer list, the acceptable values will be from 1-30.
 
 Examples:
-* `vlist` followed by `vdelete 2` deletes the 2nd volunteer in the volunteer list.
+* `vlist`, followed by `vdelete 6` will remove the 6th volunteer displayed in the volunteer list.
 * `vfind Betsy` followed by `vdelete 1` deletes the 1st volunteer in the results of the `vfind` command.
 
-### Deleting an event : `edelete`
-
-Deletes the event from the event list.
+### Clearing all volunteer entries: `vclear`
 
-Format: `edelete EVENT_ID`
-
-* Deletes the event at the specified `id`.
-* The id refers to the index number shown in the displayed event list.
-* The id **must be a positive integer** 1, 2, 3, …​
+Clears all volunteers from the volunteer list.
 
-Examples:
-* `elist` followed by `edelete 2` deletes the 2nd event in the event list.
-* `efind Beach cleaning` followed by `edelete 1` deletes the 1st event in the results of the `find` command (tentative feature)
+Format: `vclear`
 
-### Removing a volunteer in an event: `eremovev` [COMING SOON]
+### Creating an event: `ecreate`
 
-Removes the specified volunteer from an event by name or id.
+Volunteer Coordinators can create new events.
 
-Format: `eremovev vid/VOLUNTEER_ID eid/EVENT_ID` or `eremovev vn/VOLUNTEER_NAME en/EVENT_NAME`
+Format: `ecreate n/EVENT_NAME r/ROLES_NEEDED… d/DATE_AND_TIME l/LOCATION dsc/DESCRIPTION [m/MATERIALS_AND_LOGISTICS_NEEDED]... [b/BUDGET]`
 
 Parameters:
-* vn/ - Volunteer name
-* vid/ - Volunteer id
-* en/ - Event name
-* eid/ - Event id
+* n/ - Event name
+* r/ - Roles needed for the event
+* d/ - Date and time of the event
+* l/ - Location of the event
+* dsc/ - Description of the event
+* m/ - Materials needed for the event
+* b/ - Budget for the event
 
 Restrictions:
-* The maximum number of characters of the event is 50.
-* The event and volunteer name entered must exist.
-* The maximum number of characters of a volunteer name is 30.
-* The id must not exceed the number of volunteers in the event and greater or equal to 0.
+* All parameters must be separated by a single space.
+* The date and time format must be exactly `DD/MM/YYYY TTTT`
+* The budget argument must be a floating point number with 2 decimal places.
 
 Examples:
-* `eremovev vid/1 eid/1`
-* `eremovev vn/John en/fundraising`
-
-### Clearing all volunteers : `vclear`
-
-Clears all volunteers from the volunteer list.
-
-Format: `vclear`
-
-### Clearing all entries : `eclear`
+* `ecreate n/food donation r/chef r/packer d/23/9/2023 1500 dsc/help food distribution m/50 potatoes b/50` creates an event with name `food donation`, roles needed `chef` and `packer`, event date `23rd September 2023, 3pm`, description `help food distribution`, materials needed `50 potatoes` and budget `$50`
```

Comment:
ps the budget argument shld be 50.00, not 50

Reply:
can update only no worries